server/scsynth/scsynth_main.cpp

```diff
@@ -44,6 +44,38 @@ inline int setlinebuf(FILE* stream) { return setvbuf(stream, (char*)0, _IONBF, 0
 
 #endif
 
+#ifdef _WIN32
+// Add a warning when Windows Defender delays scsynth boot by 60+ seconds
+// cf. github issue #4368
+#    include <iostream>
+#    include <thread>
+#    include <mutex>
+#    include <chrono>
```

Comment:
please place these above with the other conditionally included headers (winsock2, vector) and add `<condition_variable>`